facia-tool/public/js/models/group.js

```diff
@@ -10,6 +10,7 @@ define([
     mediator
 ) {
     mediator = mediator.default;
+    copiedArticle = copiedArticle.default;
```

Comment:
I'm in the process of rewriting all of them, but I'm doing in steps... It takes time